app/api/social_links.py

```diff
@@ -39,6 +39,7 @@ def before_create_object(self, data, view_kwargs):
             event = self.session.query(Event).filter_by(id=view_kwargs['id']).one()
             data['event_id'] = event.id
 
+    view_kwargs = True
```

Comment:
This is needed when you remove the /event_copyright endpoint as given in the [docs](http://flask-rest-jsonapi.readthedocs.io/en/latest/resource_manager.html#resourcelist):

> view_kwargs:	if you set this flag to True view kwargs will be used to compute the list url. If you have a list url pattern with parameter like that: /persons/<int:id>/computers you have to set this flag to True

You can see a similar line in the [last pull request](https://github.com/fossasia/open-event-orga-server/pull/3721/files#diff-256d8e06babe7f96e6b5af3457d1bf45R70) by @magdalenesuo as well. Took reference from there.